ktor-client/ktor-client-features/ktor-client-json/common/src/io/ktor/client/features/json/JsonFeature.kt

```diff
@@ -66,10 +66,34 @@ class JsonFeature internal constructor(
          */
         @KtorExperimentalAPI
         var acceptContentTypes: List<ContentType> = listOf(ContentType.Application.Json)
+            @Deprecated(
+                "Use accept() function instead.",
+                replaceWith = ReplaceWith("accept(newList)")
+            )
             set(newList) {
                 require(newList.isNotEmpty()) { "At least one content type should be provided to acceptContentTypes" }
                 field = newList
             }
+
+        /**
+         * Internal flag that means that [accept] method has been already called.
+         */
+        private var acceptAlreadyCalled = false
+
+        /**
+         * Adds accepted content types. Be aware that [ContentType.Application.Json] accepted by default is removed from
+         * the list if you use this function to provide accepted content types.
+         * It also affects `Accept` request header value.
+         */
+        @KtorExperimentalAPI
+        fun accept(vararg contentTypes: ContentType) {
+            @Suppress("DEPRECATION")
+            if (!acceptAlreadyCalled) {
```

Comment:
```
acceptAlreadyCalled = true
```

Reply:
Looks like a test is required for those 3 cases :)

Reply:
Your truth, I added the tests :)